docker/docker-compose.prod.yml

```diff
@@ -47,6 +48,7 @@ services:
       RAVEN_DSN: $RAVEN_DSN
       ACTIONSTEP_CLIENT_ID: $ACTIONSTEP_PROD_CLIENT_ID
       ACTIONSTEP_CLIENT_SECRET: $ACTIONSTEP_PROD_CLIENT_SECRET
+      MAILCHIMP_API_KEY: $MAILCHIMP_API_KEY
```

Comment:
do you see how for Actionstep secrets the names are different for prod and test? eg:

`ACTIONSTEP_CLIENT_SECRET: $ACTIONSTEP_PROD_CLIENT_SECRET`

and

`ACTIONSTEP_CLIENT_SECRET: $ACTIONSTEP_TEST_CLIENT_SECRET`

can you please also do this for the mailchimp secrets in prod and staging

Reply:
done